src/mlpack/methods/ann/dists/normal_distribution_impl.hpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file normal_distribution_impl.hpp
+ * @author xiaohong ji
+ * @author Nishant Kumar
+ *
+ * Implementation of the Normal distribution class.
```

Comment:
```suggestion
 * Implementation of the Normal Distribution class.
```

To be consistent, with what was used in the `normal_distribution.hpp` file.